src/Tizen.Multimedia.Vision/MediaVision/InferenceType.cs

```diff
@@ -54,4 +62,43 @@ public enum InferenceTargetType
         /// </summary>
         Custom
     }
+
+    /// <summary>
+    /// Specifies the device of target. It's used for running inference backend.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public enum InferenceTargetDevice
```

Comment:
Did you intend to use this enum for flag?
If so, please add `Flag` attribute.

Reply:
This change is to support hybrid target in future. Currently the hybrid (cpu & gpu) can't be supported. So, i keep this change and  will add Flag attribute later.